test/test_nn.py

```diff
@@ -6513,6 +6513,178 @@ def test_triplet_margin_loss_swap_no_reduce(self):
         self.assertEqual(F.triplet_margin_loss(input1, input2, input3, swap=True, reduction='none'),
                          loss_reference_fns['TripletMarginLoss'](input1, input2, input3, swap=True, reduction='none'))
 
+    def test_triplet_margin_loss_with_distance_parity(self):
```

Comment:
Also see how tripletmarginloss is tested here:
https://github.com/pytorch/pytorch/blob/4ae832e1060c72cb89de1d9693629783dbe0c9a6/torch/testing/_internal/common_nn.py#L3661

Reply:
That's fair. It would detect "implementation drift" but I agree it doesn't seem worthwhile.